apps/04TowersOfHanoi.js

```diff
@@ -43,6 +45,7 @@ function checkForWin() {
 
 function towersOfHanoi(startStack, endStack) {
    if (isLegal(startStack, endStack)){
+
      movePiece(startStack, endStack);
      return true;
```

Comment:
Thank you! I see you in class tonight.

Sent from my iPhone

> On Aug 24, 2016, at 1:05 PM, maddiesolz <notifications@github.com> wrote:
> 
> In apps/04TowersOfHanoi.js:
> 
> > @@ -43,6 +45,7 @@ function checkForWin() {
> >  
> >  function towersOfHanoi(startStack, endStack) {
> >     if (isLegal(startStack, endStack)){
> > +
> >       movePiece(startStack, endStack);
> >       return true;
> Good work on the app! I will have to run this app with you during class with mocha and node, as I am not sure about the above statement. I will give you full credit for now
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>